src/components/Advanced.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+const LosslessJSON = require('lossless-json');
```

Comment:
I just realized this won't actually work without proper import syntax, can you update this please?

Reply:
I'm pretty new to this tech stack. I tried `import LosslessJSON from 'lossless-json';` and it was throwing null errors.

I'm open to suggestions.

Reply:
Read up on ES6 modules. Figured out what I needed to do based on the exports for `lossless-json`.